index.js

```diff
@@ -10,6 +10,9 @@ server.use(express.json());
 
 let db = require('./data/db');  // const or let???
 
+// QUESTION: ARE THE ENTRY IDs SUPPOSED TO AUTOMATICALLY UPDATE WHEN I PERFORM THESE CRUD OPERATIONS, OR DO I HAVE TO CODE THAT MYSELF? HOW TO DO?
+// DO I HAVE TO CODE SOMETHING SPECIFIC TO "CANCEL" THE REQUEST?
```

Comment:
Your cancel request is your catch statement or your conditional that returns an error. So if it hits one of those then the request will be cancelled and an error will be returned.